plugins/bufbuild/connect-web/v0.2.1/buf.plugin.yaml

```diff
@@ -2,7 +2,7 @@ version: v1
 name: buf.build/bufbuild/connect-web
 plugin_version: v0.2.1
 source_url: https://github.com/bufbuild/connect-web
```

Comment:
Understood - the same could be said for almost all of the URLs. I think having a link to the top-level project should be ok - otherwise using something like the NPM package URL would be good as it is unlikely to change. This gets more complicated for plugins like https://github.com/grpc/grpc/blob/master/src/compiler/cpp_generator.cc / https://github.com/apple/swift-protobuf/tree/main/Sources/protoc-gen-swift / etc.